app/assets/javascripts/templates/story.jst.ejs

```diff
@@ -20,16 +24,53 @@
   <a class="icon icons-throbber" title="<%= I18n.t('saving') %>"></a>
   <% } %>
   <span class="popover-activate">
-    <span class="icon icons-<%= story.get('story_type') %> story_type" title="<%= story.story_type %>: <%= story.id %>">
+
+
+    <% if (story.get('story_type') == 'feature') { %>
+      <span class="fa-icon-overlay">
+        <i class="fa fa-star"></i>
+        <i class="fa fa-star-o"></i>
+      </span>
+    <% } %>
+    <% if (story.get('story_type') == 'chore') { %>
+      <i class="fa fa-cog"></i>
+    <% } %>
+    <% if (story.get('story_type') == 'bug') { %>
+      <i class="fa fa-bug"></i>
+    <% } %>
+
+    <!-- TODO make it accessible
       <%= story.story_type %>
-    </span>
+
+      <% if (story.estimated()) { %>
+        <span class="estimate estimate_<%= story.get('estimate') %>">
+          <%= story.get('estimate') %>
+        </span>
+      <% } %>
+    -->
+
     <% if (story.estimated()) { %>
-    <span class="estimate estimate_<%= story.get('estimate') %>">
-      <%= story.get('estimate') %>
-    </span>
+      <span class="estimate estimate_<%= story.get('estimate') %>">
+        <% if (story.get("estimate") == 1) {  %> |  <% } %>
```

Comment:
Why not use `'|' * story.get("estimate")`?  :)
Please note that I am no longer using fulcrum, so feel free to fork the fork and modify it as you wish.